client/contract_example/cs/src/ExampleContract.cs

```diff
@@ -44,7 +44,7 @@ public ContractDataHandle Handle
                         this.Inner.Value.p
                     )
                 );
-                return new ContractDataHandle(p);
+                return new ContractDataHandle(p, OwnershipSemantics.SharedRef);
```

Comment:
Should be `Owned`: https://github.com/0xTELEPORTAL/constellation/blob/2f90dae624ebb56d001bc0f52674a2311c89149a/client/contract_example/rust/src/lib.rs#L66